Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -121,8 +121,13 @@ def format(self, record):
 # FIXME: We should decide whether we want to make this specific to web-platform-tests or to make it generic to any git repository containing tests.
 def parse_args(args):
     description = """
-To import a web-platform-tests test suite named xyz, use: 'import-w3c-tests web-platform-tests/xyz'.
-To import a web-platform-tests suite from a specific folder, use 'import-w3c-tests xyz -s my-folder-containing-web-platform-tests-folder'"""
+To import a web-platform-tests test suite named xyz, use:
+    import-w3c-tests web-platform-tests/xyz
```

Comment:
You also want tip of tree, right? 

```suggestion
To import a web-platform-tests test suite named xyz, use:
    import-w3c-tests -t web-platform-tests/xyz
```

Reply:
done. Thanks.